.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,49 @@
+---
+
+name: 🐞 Bug Report
+about: Create a report to help us improve
+title: 🐞 <title>
```

Comment:
Emoji in title adds visual noise and is redundant with label

Reply:
Agreed